src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -229,8 +229,8 @@ case class ClusterErrorDetails(code: Int, message: Option[String])
 object ClusterInitValues {
   val serviceAccountCredentialsFilename = "service-account-credentials.json"
 
-  def apply(googleProject: GoogleProject, clusterName: ClusterName, bucketName: GcsBucketName, clusterRequest: ClusterRequest, dataprocConfig: DataprocConfig,
-            clusterFilesConfig: ClusterFilesConfig, clusterResourcesConfig: ClusterResourcesConfig, proxyConfig: ProxyConfig, swaggerConfig: SwaggerConfig,
+  def apply(userEmail: WorkbenchEmail, googleProject: GoogleProject, clusterName: ClusterName, bucketName: GcsBucketName, clusterRequest: ClusterRequest, dataprocConfig: DataprocConfig,
```

Comment:
nit: can you rename this to `userEmailLoginHint` or something? it took me ages to figure out where it was going and where the login hint came from, because neither the name nor its position in the parameter list was the same

Reply:
:) good call, will do.